models/Posting.js

```diff
@@ -29,14 +29,17 @@ const PostingSchema = Schema({
     type: [String],
   },
   requestedItems: [String],
-  // imgSrc: {
-  //   data: Buffer,
-  //   contentType: String,
-  // },
+  images: [String],
   ownerId: {
     type: String,
     required: true,
   },
+  ownerUsername: String, 
```

Comment:
should we keep this information on the user-level and access through api call? reason being if the user changes their user name, the username on the posting will not be updated.

Reply:
Yeah username could be pulled out and accessed from a get user api call. Since this is only used in the item details view, we'd also need the api call to get the reviews from the user as well. Will change this